bitnami/nginx-ingress-controller/Chart.yaml

```
@@ -26,4 +26,4 @@ name: nginx-ingress-controller
 sources:
   - https://github.com/bitnami/bitnami-docker-nginx-ingress-controller
   - https://github.com/kubernetes/ingress-nginx
-version: 7.4.9
+version: 7.5.0
```

You're not adding any new feature. Also, it doesn't affect the default configuration of the chart.
Therefore, I'd bump the patch version instead:
```suggestion
version: 7.4.10
```